tests/resources/scripts/test-metric.lua

```diff
@@ -0,0 +1,17 @@
+---
+--- Generated by Luanalysis
+--- Created by Soukaina.
+--- DateTime: 19/06/2023 12:17
+---
+-- Init function
+function init(params)
+    broker_log:set_parameters(1, "/tmp/test4.log")
+end
+
+-- Write function
+function write(d)
+    if d._type == 196617 then
+            broker_log:info(0, "name: " .. tostring(d.name) .. " corresponds to metric id " .. tostring(d.metric_id))
+    end
+    return true
+end
```

Comment:
Once the file creation is moved inside the robot file, please remove this one.

Reply:
deleted